### PR TITLE
Add pyLoad

### PR DESCRIPTION
**Pull request by ripps818**
I'm not super familiar with git and pull requests, so I'm sorry if I've made any mistakes.

Pyload is simple web-interface download manager that is able to handle most of the major download services online. I think I covered all the bases and added it to nas.yml and traefik.toml as needed.

I tested this in a VM running Ubuntu 18.04 and everything seems to work.

### CODE REVIEW

File: group_vars/all.yml.dist [hunk 1]
```diff
@@ -275,14 +278,21 @@ heimdall_data_directory: "{{ docker_home }}/heimdall"
 ### Transmission
 ###
 transmission_available_externally: "false"
-transmission_with_openvpn_available_externally: "false"
```
Comment (caillou) [inline group_vars/all.yml.dist]: Why did you remove this line?

**Comment by ripps818** [comment]
Oh that must have been an accident, I copy-pasted a bunch of stuff from my
own configs.

**Comment by davestephens** [comment]
Hi, please can you fix the b0rked build then I can take a look at merging this? You'll also need to ad some docs, see the other application doc pages for examples.

**Comment by davestephens** [comment]
This is awesome, thank you. Sorry it's taken me a while to re-reviewing it!

**Comment by davestephens** [comment]
Oh and congrats on your first ever merged PR :D

**Comment by xerosanyam** [comment]
@ripps818 the defaule pyload config is not working for me. getting 
image
any ideas on how to fix this?

**Comment by ripps818** [comment]
@xerosanyam No idea. You might want to hit up the pyload developers and see if they know. Only they would know what an error 36 means.
